test/print-routes.test.js

```diff
@@ -22,6 +22,21 @@ test('should print routes', async t => {
   t.ok(spy.calledWithMatch('debug', '└── / (GET|POST)\n'))
 })
 
+test('should print routes via cli', t => {
+  t.plan(2)
+
+  const spy = sinon.spy()
+  const command = proxyquire('../print-routes', {
+    './log': spy
+  })
+  command.cli(['./examples/plugin.js'])
```

Comment:
Could you add a test with the `--help`?

Reading the code, this case should not return a promise, so the `.then` should break.
So maybe it is necessary to move the `if(fastify)` in the upperscope

Reply:
I'll look in detail later but from what I've seen the `help` case uses explicit `process.exit` so that `.then` will never execute.

Reply:
You are right!

https://github.com/fastify/fastify-cli/blob/056c0b510b27d886ddb9c68be63fca42fc37b935/util.js#L12

(in this case, the `if` statement should not be necessary)